src/MongoDB/Monitoring/CommandFailedEvent.c

```diff
@@ -249,7 +249,11 @@ static HashTable* php_phongo_commandfailedevent_get_debug_info(phongo_compat_obj
 	if (intern->has_service_id) {
 		zval service_id;
 
-		phongo_objectid_init(&service_id, &intern->service_id);
+		if (!phongo_objectid_new(&service_id, &intern->service_id)) {
+			/* Exception should already have been thrown */
+			goto done;
```

Comment:
I noticed you didn't check the return value of `phongo_objectid_new` earlier in the file. Is there any reason to do so here? Looking at the implementation it always returns `true` so I think we can forgo the conditional.

Likewise for other calls to `phongo_objectid_new` and any other similar "new" methods.

Reply:
SGTM. I overlooked that the other instances were at the end of a function where the check would have made no difference.